devito/data/utils.py

```diff
@@ -389,7 +389,7 @@ def flip_idx(idx, decomposition):
                 stop = i.stop
             processed.append(slice(start, stop, i.step))
         else:
-            processed.append(i)
+            processed.append(slice(i, i+1, 1))
```

Comment:
`1` or `None`?

Reply:
Doesn't matter I don't think? `1` is just more 'explicit'?